Cannot find module '../../core/dist' error with threads-database

Getting the following when trying to compile a project that depends on @textile/threads-database@0.2.2:

```
ERROR in node_modules/@textile/threads-database/dist/db.d.ts:65:34 - error TS2307: Cannot find module '../../core/dist'.

65     ownLogInfo(): Promise<import("../../core/dist").LogInfo | undefined>;
```

The d.ts file seems to reference an incorrect path outside of its npm package.

--- a/packages/database/src/db.ts
+++ b/packages/database/src/db.ts
@@ -12,6 +12,7 @@ import {
   Multiaddr,
   Identity,
   Libp2pCryptoIdentity,
+  LogInfo,
 } from '@textile/threads-core'
 import LevelDatastore from 'datastore-level'
 import { EventBus } from './eventbus'
@@ -121,7 +122,7 @@ export class Database extends EventEmitter2 {
   }
 
   @Cache({ duration: 1800000 })
-  async ownLogInfo() {
+  async ownLogInfo(): Promise<LogInfo | undefined> {
     return this.threadID && this.network.getOwnLog(this.threadID, false)
   }
 

